ldm/invoke/generator/inpaint.py

```diff
@@ -202,12 +204,17 @@ def get_make_image(self,prompt,sampler,steps,cfg_scale,ddim_eta,
             # Do infill
             if infill_method == 'patchmatch' and PatchMatch.patchmatch_available():
                 init_filled = self.infill_patchmatch(self.pil_image.copy())
-            else: # if infill_method == 'tile': # Only two methods right now, so always use 'tile' if not patchmatch
+            elif infill_method == 'tile':
                 init_filled = self.tile_fill_missing(
                     self.pil_image.copy(),
                     seed = self.seed,
                     tile_size = tile_size
                 )
+            elif infill_method == 'solid':
+                solid_bg = PIL.Image.new("RGBA", init_image.size, (0x7F, 0x7F, 0x7F, 0xFF))
```

Comment:
I agree with @psychedelicious on this. Could the solid color be exposed as a parameter passed to `get_make_image()`, defaulting to grey?

Reply:
I implemented the change so that we can move this along.